src/segment-loader.js

```diff
@@ -1224,6 +1224,11 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     const Cue = window.WebKitDataCue || window.VTTCue;
     const value = {
+      bandwidth: segmentInfo.playlist.attributes['BANDWIDTH'],
```

Comment:
should we not rather fix the fact that `attributes` may be `undefined`? it could be initialized `{}` by default?

sure, we can go for dot notation :)

and how about we just add `attributes` as such to the cue data? that way the user has all the info, no need to make this here a replication of the semantic that is already in attributes with lots of checks and so on. Because the attributes themselves may not be present in some playlists. It's just about making available all the info we have I guess :)